visidata/mouse.py

```diff
@@ -73,12 +73,12 @@ def parseMouse(vd, **kwargs):
 
 @VisiData.api
 def handleMouse(vd, sheet):
+    vd.keystrokes = ''
```

Comment:
Is it necessary to move all this code outside the `try` block?  I think it's probably harmless, but does increase the diff size.